src/services/validatorSchema.ts

```diff
@@ -0,0 +1,17 @@
+import * as yup from 'yup';
+
+export const inventoryItemSchema = yup.object({
+    user_id: yup.string().typeError("user_id must be a string").defined("The user_id must be provided, don't forget it ! "),
```

Comment:
thanks, your validators look good. 
2 things that can be improve:
- the error message can be more explicit and include the expected type (the error the user will see is that the field should be provided, but it doesn't say what the type is, so I'd have to go back to the documentation. Putting the type will save some time and frustration :) something like: `The user_id (string) must be provided.`
- also, I'm revisiting my idea that providing a number and accepting yup to convert to string may not be a good idea. If I supply a number for the `user_id` string it should fail instead of accepting it. Let's be strict if you don't mind, and throw an error when user_id is not a string